src/ProjectOrigin.WalletSystem.Server/Models/CertificateViewModel.cs

```diff
@@ -1,15 +1,14 @@
+using Google.Protobuf.WellKnownTypes;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Google.Protobuf.WellKnownTypes;
-using ProjectOrigin.Common.V1;
 
 namespace ProjectOrigin.WalletSystem.Server.Models;
 
 public record CertificateViewModel
 {
     public required Guid Id { get; init; }
-    public required string Registry { get; init; } = string.Empty;
+    public required string RegistryName { get; init; } = string.Empty;
     public required GranularCertificateType CertificateType { get; init; }
     public required string GridArea { get; init; } = string.Empty;
```

Comment:
string.Empty should probably removed, don't even know why they are there, will you are changing this.